carball/analysis/stats/boost/boost.py

```diff
@@ -85,13 +85,14 @@ def get_average_boost_level(player_dataframe: pd.DataFrame) -> np.float64:
 
     @classmethod
     def get_num_stolen_boosts(cls, player_dataframe: pd.DataFrame, is_orange):
-        big_pads_collected = player_dataframe[player_dataframe.boost_collect == True]
-        if is_orange == 1:
-            boost_collected_in_opposing_third = big_pads_collected[cls.field_constants.get_third_0(big_pads_collected)]
+        big = cls.field_constants.get_big_pads()
+        # Get big pads below or above 0 depending on team
+        if is_orange:
+            opponent_pad_labels = big[big[:, 1] < 0][:, 2]
```

Comment:
can you add a bit more detailed explanation of how this logic works?

It is a bit hard to read